src/bundle.ts

```diff
@@ -0,0 +1,25 @@
+import { get, set } from 'lodash';
+
+import { pointerToPath } from './pointerToPath';
+import { traverse } from './traverse';
+
+export const bundleTarget = ({ root, cur, path = '#/__target__' }: { root: any; cur?: any; path?: string }) => {
```

Comment:
I'd rename `root` to `document`, and remove the default `path` property (force caller to set it explicitly so less magic happening under the hood).

Lastly, would move `cur` to 2nd argument, since it's not really something end user should pass in. Would also make things `unknown` unless that turns out to be a pain.

`export const bundleTarget = ({ document, path }: { document: unknown; path: string }, cur?: unknown) => {`

Reply:
Word!